cluster/vagrant/util.sh

```diff
@@ -288,6 +303,11 @@ function restart-kube-proxy {
   ssh-to-node "$1" "sudo systemctl restart kube-proxy"
 }
 
+# Restart the apiserver
+function restart-apiserver {
```

Comment:
Expediency just. Let this fly and I will port this across so that services e2e looks cleaner also.